tools/XCConfigExporter/main.swift

```diff
@@ -0,0 +1,66 @@
+import XcodeCompilationDatabaseCore
+import Foundation
+import ShellOut
+
+guard CommandLine.arguments.count > 1 else {
+    print("""
+          usage: /path/to/xcconfig
+          """)
+    exit(0)
+}
+
+/// This program works by doing a build of an iOS application with a user
+/// specified xcconfig, and exporting the flags that Xcode passed to various
+/// compilers
+do {
+    guard let fixtureBundle = Bundle.main.path(forResource: "Fixtures", ofType:
+                                             "bundle") else {
+        fatalError("Missing fixtures")
+    }
+    let xcconfig = CommandLine.arguments[1]
+
+    // Create a temp directory for the template project, then copy it over
+    let attributes: [FileAttributeKey: Any] = [FileAttributeKey.posixPermissions: 0o777]
+    let tmpDir = NSTemporaryDirectory() + "/" + UUID().uuidString
```

Comment:
I think there is a string api to add a path component which will do the "/" concatenation for you

Reply:
Good point. Also, we currently use the `Path` lib in XCHammer, it's now easy to pull in the same deps if we want